### DISCUSSION TITLE
Prevent users of my static website from accessing my images folder.

### DISCUSSION THREAD

**Discussion by lluvanir** [vercel/community#4877] [Help] [answered]
### Summary

I need something equivalent to .htaccess for Vercel; I want users of my site to be prevented from accessing the folder where my images and CSS are located.
Which feature of vercel.json can I use for this?
Would .vercelignore also be a valid choice? I'm afraid to use it and end up not loading both the images and the CSS.
Sorry, I am not knowledgeable in these matters.

### Steps to Reproduce

users can simply have full access to my image gallery.

**Comment by amyegan** [comment, marked as answer]
Hi @lluvanir. I think I'm missing some context. Since images and CSS are served to the browser, they are generally not secret or private.

Can you share a bit more about why you want to prevent access? If you share the reason, someone here might know a suitable solution.

**Reply by lluvanir** [reply]
Thank you for responding, I just wanted to make it difficult for people to easily copy my css code or access the link where all the images from my website are gathered, all through vercel.json. But now I understand. Thanks
